broker/broker-connection-factory/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,61 @@
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
+           http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <ext:property-placeholder placeholder-prefix="$[" placeholder-suffix="]"/>
+
+    <reference id="encryptionService" interface="ddf.security.encryption.EncryptionService"/>
+
+    <!--
+     This is a common connection factory if a service needs specific options, it's best not to
+     modify this and instead create its own instance of the factory. Its also good to note that this
+     is NOT a pooled connection factory which could cause starvation issues if a consumer and producer
+     were sharing the same pool.
+     -->
+
+    <bean id="activeMQJMSClientFactoryWrapper"
+          class="org.codice.ddf.broker.config.ActiveMQJMSClientFactoryWrapper">
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.broker.config.ActiveMQJMSClientFactoryWrapper"
+                update-strategy="component-managed" update-method="update"/>
+        <argument ref="encryptionService"/>
+        <property name="url"
+                  value="(vm://0,tcp://$[artemis.live.host]:$[artemis.multiprotocol.port],tcp://$[artemis.backup.host]:$[artemis.multiprotocol.port])?ha=true;sslEnabled=false;keyStorePath=$[javax.net.ssl.keyStore];keyStorePassword=$[javax.net.ssl.keyStorePassword];enabledCipherSuites=$[https.cipherSuites];enabledProtocols=$[https.protocols]"/>
+        <property name="username" value="admin"/>
+        <property name="password" value="admin"/>
+    </bean>
+
+    <bean id="jmsConnectionFactory" factory-ref="activeMQJMSClientFactoryWrapper"
+          factory-method="createConnectionFactory"/>
+
+    <service interface="javax.jms.ConnectionFactory" ref="jmsConnectionFactory">
+        <service-properties>
+            <entry key="id" value="jmsConnectionFactory"/>
+            <entry key="name" value="jmsConnectionFactory"/>
+            <entry key="protocol" value="Core"/>
+            <entry key="jmsVersion" value="2.0"/>
+            <entry key="host" value="localhost"/>
```

Comment:
they are for service look up i don't think any of the ones below jmsVersion make sense anymore since we ended up making them configurable